server/storage.js

```diff
@@ -0,0 +1,208 @@
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+
+const conf = require('./config.js');
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const crypto = require('crypto');
+
+let isProduction =
+  conf.env === 'production' &&
+  conf.s3_bucket !== 'localhost' &&
+  conf.bitly_key !== 'localhost';
+
+if (isProduction) {
+  module.exports = {
+    length: AWSLength,
+    get: AWSGet,
+    set: AWSSet,
+    delete: AWSDelete,
+    forceDelete: AWSForceDelete
+  };
+} else {
+  module.exports = {
+    length: LocalLength,
+    get: LocalGet,
+    set: LocalSet,
+    delete: LocalDelete,
+    forceDelete: LocalForceDelete
+  };
+}
+
+function LocalLength(id) {
```

Comment:
Traditionally, in js this case style is used for constructors. Let's use camelCase for these function names